src/pages/settings/AppDownloadLinks.tsx

```diff
@@ -1,75 +1,75 @@
-import React from 'react';
+import React, {useRef} from 'react';
 import {ScrollView} from 'react-native';
-import _ from 'underscore';
+import type {View} from 'react-native';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import * as Expensicons from '@components/Icon/Expensicons';
 import MenuItem from '@components/MenuItem';
+import type {MenuItemProps} from '@components/MenuItem';
 import ScreenWrapper from '@components/ScreenWrapper';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import Navigation from '@libs/Navigation/Navigation';
 import * as ReportActionContextMenu from '@pages/home/report/ContextMenu/ReportActionContextMenu';
 import * as Link from '@userActions/Link';
 import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ROUTES from '@src/ROUTES';
 
-const propTypes = {
-    ...withLocalizePropTypes,
-    ...windowDimensionsPropTypes,
+type DownloadMenuItem = MenuItemProps & {
+    translationKey: TranslationPaths;
+    openAppDownloadLink: () => void;
+    downloadLink: string;
 };
 
-function AppDownloadLinksPage(props) {
+function AppDownloadLinksPage() {
     const styles = useThemeStyles();
-    let popoverAnchor;
+    const {translate} = useLocalize();
+    const popoverAnchor = useRef<View>(null);
 
-    const menuItems = [
+    const menuItems: DownloadMenuItem[] = [
         {
             translationKey: 'initialSettingsPage.appDownloadLinks.android.label',
-            icon: Expensicons.Android,
-            iconRight: Expensicons.NewWindow,
-            action: () => {
+            openAppDownloadLink: () => {
                 Link.openExternalLink(CONST.APP_DOWNLOAD_LINKS.ANDROID);
             },
-            link: CONST.APP_DOWNLOAD_LINKS.ANDROID,
+            downloadLink: CONST.APP_DOWNLOAD_LINKS.ANDROID,
```

Comment:
Any reason for renaming? action > openAppDownloadLink, link > downloadLink

Reply:
Yes, I was following my understanding of the Style document in the Naming Conventions->Event Handlers section found [here](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#naming-conventions). I then adjusted link to downloadLink to be more descriptive. Please let me know if 'action' and 'link' are preferred to the more descriptive names.

Reply:
original names are fine to me